frontend/packages/console-app/src/components/guided-tours/GuidedTourItem.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { CatalogTile } from '@patternfly/react-catalog-view-extension';
+import { GuidedTourItem, TourStatus } from './utils/guided-tour-typings';
+import TourItemHeader from './TourItemHeader';
+import TourItemDescription from './TourItemDescription';
+import TourItemFooter from './TourItemFooter';
+import './GuidedTourItem.scss';
+
+type GuidedTourItemProps = GuidedTourItem & TourStatus;
+
+const GuidedTourItem: React.FC<GuidedTourItemProps> = ({
+  iconURL,
+  altIcon,
+  name,
+  description,
+  status,
+  active,
+  duration,
+  prerequisites,
+}) => (
+  <CatalogTile
+    iconImg={iconURL}
+    iconAlt={altIcon}
+    className="odc-guided-tour-item"
+    featured={active}
+    title={<TourItemHeader name={name} status={status} duration={duration} />}
+    description={<TourItemDescription description={description} prerequisites={prerequisites} />}
```

Comment:
I raised PF issue: https://github.com/patternfly/patternfly-react/issues/4451

Because this description doesn't work correctly without needing custom css to break out of the default layout.